ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_paddle_activation.py

```diff
@@ -436,6 +437,50 @@ def test_paddle_tanhshrink(
     )
 
 
+@st.composite
+def _get_input_and_new_shape(draw):
+    new_shape = draw(
+        helpers.get_shape(
+            min_num_dims=2, max_num_dims=10, min_dim_size=2, max_dim_size=10
+        ).filter(lambda x: x[0] % 2 == 0 and x[1] % 2 == 0 and x[len(x) - 1] % 2 == 0)
+    )
+    x_dtype, x = draw(
+        helpers.dtype_and_values(
+            available_dtypes=helpers.get_dtypes("valid"),
+            min_num_dims=2,
+            max_num_dims=10,
+            min_dim_size=2,
+            max_dim_size=10,
+            shape=new_shape,
+        )
+    )
+    return x_dtype, x
+
+
+@handle_frontend_test(
+    fn_tree="paddle.nn.functional.glu",
+    input_x_shape=_get_input_and_new_shape(),
+    test_with_out=st.just(True),
```

Comment:
why are you forcing the function to test the out argument although it doesn't exist?